### DISCUSSION TITLE
Add search in commits

### DISCUSSION THREAD

**Discussion by VadimMuhtarov** [community/community#17903] [Other Feature Feedback, Questions, & Ideas]
Please add the ability to search in commits

**Comment by kimdwkimdw** [comment]
https://cs.github.com

**Reply by lmstearn** [reply]
This is not to be confused with searching for [commit messages](https://docs.github.com/en/search-github/searching-on-github/searching-commits#search-within-a-users-or-organizations-repositories).
One of the most popular used search types in one's own repository: in what commit was such and such string first written, in how many commits it remained, and by deduction, what commit was it removed?
Thanks for the waitlist option, for me, the github search has so far been reliable, it has to be noted that in AutoHotkey (still not an official language) repos, larger script files (over .5 mb  containing some binary data) don't yield search hits as the Github search engine filters them out.  Otherwise, _search_term_ **user:**_username_ in [search](https://github.com/search) works perfectly!
